app/models/pool.ts

```diff
@@ -105,6 +105,43 @@ export class Pool extends PoolRecord {
 
         this.tags = ModelUtils.tagsFromMetadata(this.metadata);
     }
+
+    public get osName(): string {
+        if (this.cloudServiceConfiguration) {
+            let osName: string;
+            let osFamily = this.cloudServiceConfiguration.osFamily;
+
+            if (osFamily === 2) {
+                osName = "Windows Server 2008 R2 SP1";
```

Comment:
We are using the same thing in the OS picker maybe we could share this structure